client/app/achievements/achievements.less

```diff
@@ -1,3 +1,7 @@
 ng-invald {
   color: red
 }
+
+.event-description > img {
+  width: 100%;
```

Comment:
An image within a link would break this and that case wouldn't break the layout. I think an image within a list would also break it, although that may cause it to blow out of the box  anyway so you can probably ignore that case.